### PR TITLE
swagger.io support for lighty.io

### PR DESCRIPTION
**Pull request by jveverka**
Signed-off-by: Juraj Veverka <juraj.veverka@pantheon.tech>

### CODE REVIEW

File: lighty-examples/lighty-community-restconf-netconf-app/pom.xml [hunk 1]
```diff
@@ -52,6 +52,11 @@
             <groupId>io.lighty.resources</groupId>
             <artifactId>log4j-properties</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.lighty.modules</groupId>
+            <artifactId>lighty-swagger</artifactId>
+            <version>${project.version}</version>
```
[the remark this answers is not in the document]
Comment (jveverka) [reply lighty-examples/lighty-community-restconf-netconf-app/pom.xml]: done

File: lighty-modules/lighty-swagger/pom.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <groupId>io.lighty.modules</groupId>
+    <artifactId>lighty-swagger</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <url>https://github.com/PantheonTechnologies/lighty-core</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.lighty.core</groupId>
+            <artifactId>lighty-controller</artifactId>
+            <version>${project.version}</version>
```
[the remark this answers is not in the document]
Comment (jveverka) [reply lighty-modules/lighty-swagger/pom.xml]: done

File: lighty-modules/pom.xml [hunk 3]
```diff
@@ -27,5 +27,6 @@
         <module>lighty-restconf-nb-community</module>
         <module>integration-tests</module>
         <module>integration-tests-aaa</module>
+        <module>lighty-swagger</module>
```
[the remark this answers is not in the document]
Comment (jveverka) [reply lighty-modules/pom.xml]: done

**Comment by rovarga** [comment]
Fixes #163

**Pull request merged**